Formula/typedb.rb

```diff
@@ -1,20 +1,31 @@
 class Typedb < Formula
-  desc "Distributed hyper-relational database for knowledge engineering"
+  desc "Strongly-typed database with a rich and logical type system"
   homepage "https://vaticle.com/"
   url "https://github.com/vaticle/typedb/releases/download/2.14.3/typedb-all-mac-2.14.3.zip"
   sha256 "41a574d4d0fafcdfd678599b488dfb3aa7e2c4664e111dbd479bdf0a4dbd12a7"
   license "AGPL-3.0-or-later"
+  revision 1
 
   bottle do
     sha256 cellar: :any_skip_relocation, all: "9d2db455c9a587a9eb74879be312ccb44aa30d4a9334c3ba1cc5d647b7557d69"
   end
 
   depends_on "openjdk@11"
 
+  def setup_directory(dir)
+    typedb_dir = var / name / dir
```

Comment:
How will typedb know to write here?

Reply:
It writes to the symlink that's created a few lines down.
```rb
    orig_dir = libexec / "server" / dir
    rm_rf orig_dir
    ln_s typedb_dir, orig_dir
```
`libexec/server/data` e.g. will point to `var/typedb/data`.

Reply:
Is there no configuration to tell typedb where to look?

Reply:
There is, but the configuration would also get overwritten on the upgrade.

Reply:
I'm starting to get the idea that typedb maybe isn't designed to be installed by a package manager. We can't tell it where to look with the wrapper script or anything?

Reply:
It's not, really. It's designed for local installs, but it somehow ended up in homebrew, users install it from here and get their data destroyed. A wrapper script would be a lot more effort than the symlink solution for very little benefit.